src/main/java/com/shade/decima/ui/data/viewer/texture/reader/ImageReader.java

```diff
@@ -38,6 +36,8 @@ public BufferedImage read(@NotNull ByteBuffer buffer, int width, int height) {
 
     protected abstract void readBlock(@NotNull ByteBuffer buffer, @NotNull BufferedImage image, int x, int y);
 
+    protected abstract BufferedImage createImage(int width, int height);
```

Comment:
Please specify nullability annotations for the method's return type. Don't forget about implementations.